src/host/readData.hpp

```diff
@@ -23,6 +23,8 @@ Revision History:
 #include "../server/IWaitRoutine.h"
 #include "../server/WaitTerminationReason.h"
 
+class InputBuffer;
+

```

Comment:
One of the test files had this weird comment about "oh you need to include InputBuffer.h here first. idk why but it fails compilation otherwise". I fixed that issue by properly forward-declaring `InputBuffer` there.